tools/github/README.md

```diff
@@ -0,0 +1,24 @@
+# GitHub Applications and Integrations
+
+This document and directory contain information about all configured GitHub
+applications and integrations with the OpenTelemetry organization.
+
+| Name             | Description                                          | Scopes                      | Notes                                                                  |
+|------------------|------------------------------------------------------|-----------------------------|------------------------------------------------------------------------|
+| [Azure Pipelines](https://github.com/apps/azure-pipelines)  | Support for Azure Pipelines                          | Org-wide                    | Included by default                                                    |
+| [ciMate](https://github.com/apps/cimate)           | TBD                                                  | TBD                         | TBD                                                                    |
```

Comment:
I don't see this(?)

Reply:
This is what I'm seeing now:

![image](https://github.com/open-telemetry/community/assets/17327289/3fde4dff-6bde-42a5-8a93-477fcf53090c)

Reply:
Anyways, I don't feel this is a blocker for this PR, as long as we merge the structure, folks can send follow up PRs to keep things up-to-date.